python/dask_cudf/setup.py

```diff
@@ -1,19 +1,14 @@
 # Copyright (c) 2019-2023, NVIDIA CORPORATION.
 
-import os
-
-import versioneer
 from setuptools import find_packages, setup
 
-cuda_suffix = os.getenv("RAPIDS_PY_WHEEL_CUDA_SUFFIX", default="")
-
 install_requires = [
     "dask>=2023.1.1",
     "distributed>=2023.1.1",
     "fsspec>=0.6.0",
     "numpy",
     "pandas>=1.0,<1.6.0dev0",
-    f"cudf{cuda_suffix}==23.4.*",
+    "cudf==23.4.*",
```

Comment:
Do we not still need to maintain the wheel suffix? Or has something else changed that means this won't be needed?

Reply:
Something else has changed. Specifically, the suffix is now added in the same `apply_wheel_modification.sh` script that also sets the release version used during wheel builds.